app/scripts/modules/core/delivery/details/executionDetails.less

```diff
@@ -159,3 +159,24 @@
     }
   }
 }
+
+.execution-details {
+  .action-buttons {
+    margin: 10px -10px 0;
+    button {
+      margin: 0 10px;
+    }
+  }
+}
+
+.popover-content {
+  .action-buttons {
+    display: flex;
+    align-items: stretch;
+    margin: 10px -5px 0;
+    button {
+      flex: 1 0 auto;
+      margin: 0 5px;
+    }
+  }
+}
```

Comment:
This change just lets buttons take up the normal amount of space in the details view, while filling the width of the popover.